.jshintrc

```diff
@@ -0,0 +1,32 @@
+{
+    "node": true,
+    "browser": true,
```

Comment:
I think this should be false. Our backend shouldn't be doing anything that's visible in the browser it's just serving data to the frontend.